e2e/device/edit-device-name.yaml

```diff
@@ -0,0 +1,60 @@
+appId: com.comapeo
+env:
+  CAT: MAIN
```

Comment:
Perhaps instead of MAIN lets keep it consistent with the folder name (so either `DEVICE` or `PROJECT_SETTINGS` depending on what you chose to use

Reply:
Folder is now project and there is now a doc for it. I left the doc overview pretty basic but will add to that in subsequent PRs as I add more Project tests. I also changed the CAT to project for some of the UI, but some is still MAIN because it is in the main part of the app if that makes sense...